cpp/scripts/format.py

```diff
@@ -0,0 +1,64 @@
+import argparse
```

Comment:
any thoughts on how we could avoid copying this between two repos? publish a docker image, or a nodejs binary on npm that can be used with npx (or just pypi), or something?